src/sevcore_linux.cpp

```diff
@@ -689,10 +690,10 @@ int SEVDevice::generate_vcek_ask(const std::string output_folder,
     // Set struct to 0
     memset(&id_buf, 0, sizeof(sev_user_data_get_id));
 
-    // Zero the memory used for the URL, just in case.
-    memset(&cmd, 0, sizeof(cmd));
-
     do {
+        cmd += "-O " + der_cert_w_path + " \"";
+        cmd += KDS_VCEK;
+        cmd += "Milan/";
 
         fmt = "wget -O %s \"%sMilan/%s?blSPL=%02d&teeSPL=%02d&snpSPL=%02d&ucodeSPL=%02d\"";
```

Comment:
```suggestion
```
If we are going to get rid of the `sprintf` below, it would makes sense to also get rid of the fmt variable (as it isn't being used anymore).